readme.md

```diff
@@ -167,6 +178,16 @@ If succeeded you should receive following JSON response with code 200:
 }
 ```
 
+#### PUT /dashboard/status - Update total case, death case, recovered case and suspect case status of Covid-19
+```
+`curl -d '{"lk_total_case": 98,"lk_recovered_case": 98,"lk_total_deaths": 99,"lk_total_suspect": 99}' -H "Content-Type: application/json" -X PUT http://localhost:8000/application/dashboard/status`

```

Comment:
no need curl command here. Leave in same format as other APIs. This is for highlevel view only. Postman is for invoking/testing